### PR TITLE
RDM-12540: Fix json issue on access control list

### PR DESCRIPTION
**Pull request by kiran-yenigala-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-12540

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/ccd/domain/model/definition/AccessControlListTest.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        accessControlList.setUpdate(false);
+        accessControlList.setRead(false);
+        accessControlList.setDelete(false);
+        accessControlList.setCreate(true);
+        accessControlList.setAccessProfile("test");
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        String value = objectMapper.writeValueAsString(accessControlList);
+
+        System.out.println(value);
```
Comment (danlysiak) [inline src/test/java/uk/gov/hmcts/ccd/domain/model/definition/AccessControlListTest.java:24]: Can we assert this is as expected instead of just logging?

**Comment by Thor-tech-of-metal** [comment]
The class RoleToAccessProfileDefinition also have json errors may be we can add the fixes in this pr, I have fixed on my env I am happy to share the classes :)

**Review by Thor-tech-of-metal — COMMENTED** [review]
The class RoleToAccessProfileDefinition also have json errors may be we can add the fixes in this pr, I have fixed on my env I am happy to share the classes :)

File: src/test/java/uk/gov/hmcts/ccd/domain/model/definition/AccessControlListTest.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+    void shouldValidateAccessControlListSerializationWithRole() throws JsonProcessingException {
```
Comment (rafalkalita) [inline src/test/java/uk/gov/hmcts/ccd/domain/model/definition/AccessControlListTest.java:27]: Missing '@Test' annotation

**Review by rafalkalita — APPROVED** [review]

**Pull request merged**